appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/runtime/deployer/ManagedScheduledExecutorDefinitionDeployer.java

```diff
@@ -185,7 +185,7 @@ public void setContextInfo(String value) throws PropertyVetoException {
 
         @Override
         public String getDescription() {
-            return null;
+            return "Managed Scheduled Executor Definition";
```

Comment:
Not sure about this change - shouldn't this be getting the value of the description property? `descriptor.getDescription()`
If the property is null then returning null (or an empty string) is probably correct, rather than a cookie cutter string like this.

Reply:
I followed the same code in ManagedExecutorDescriptorDeployer.java. AFAIK this has no impact on anything.

Reply:
probably this value can be defined on a constant variable

Reply:
Actually, after reviewing the code, it really makes sense to use the descriptor's value. 
Updated.
Thanks!